spec/support/vcr.rb

```diff
@@ -1,21 +1,21 @@
 VCR.configure do |c|
```

Comment:
My suggestion is to throw VCR out the window and just stub the request. How many cassettes though are we creating/need to create?